easybuild/easyconfigs/v/Visit/Visit-3.4.1-foss-2023a.eb

```diff
@@ -0,0 +1,75 @@
+easyblock = 'CMakeMake'
+
+name = 'Visit'
+version = '3.4.1'
+
+homepage = 'https://github.com/visit-dav/visit'
+description = """
+VisIt is an Open Source, interactive, scalable, visualization, animation
+and analysis tool. From Unix, Windows or Mac workstations, users can
+interactively visualize and analyze data ranging in scale from small
+(<101 core) desktop-sized projects to large (>105 core) leadership-class
+computing facility simulation campaigns. Users can quickly generate
+visualizations, animate them through time, manipulate them with a
+variety of operators and mathematical expressions, and save the
+resulting images and animations for presentations. VisIt contains a rich
+set of visualization features to enable users to view a wide variety of
+data including scalar and vector fields defined on two- and
+three-dimensional (2D and 3D) structured, adaptive and unstructured
+meshes. Owing to its customizeable plugin design, VisIt is capabable of
+visualizing data from over 120 different scientific data formats.
+"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+sources = [{
+    'filename': 'v%(version)s.tar.gz',
+    'git_config': {
```

Comment:
@pavelToman Can we avoid this and use the prepared https://github.com/visit-dav/visit/releases/download/v3.4.1/visit3.4.1.tar.gz as source tarball instead?

Reply:
After the change of sources to visit3.4.1.tar.gz there are some problem. It seems it needs download git recursively to build it by CMakeMake: https://visit-sphinx-github-user-manual.readthedocs.io/en/stable/building_visit/Building_Directly_With_CMake.html#

The `git lfs` is also required for reg. tests: https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/dev_manual/test_suite.html#a-note-about-git-lfs

Reply:
Should I add git-lft as a dependency?

Reply:
I tried to install Visit with `visit3.4.1.tar.gz` but without reg. tests and it seems it is ok - I tested it by run gui on desktop cluster with `visit`